patches/mysql2+^3.0.0-rc.1.patch

```diff
@@ -145,44 +145,44 @@ index 47970e9..a0fec80 100644
 -          throw new TypeError(
 -            'Bind parameters must not contain undefined. To pass SQL NULL specify JS null'
 -          );
-+          val = null
++          val == null;
```

Comment:
Why is this now checking `val == null`? undefined is reassigned to null because of nil parameters from Lua.